packages/ton-client/src/style.css

```diff
@@ -37,3 +33,11 @@ body {
   box-shadow: 0 0 1.5rem #0002;
   text-shadow: 0 0 0.5rem #0002;
 }
+
+#camera {
+  position: absolute; 
+}
+
+* {
```

Comment:
outline for everything?

Reply:
upsi! That's a trick that I use for debugging layout issues. 
It's great, because you directly see the boxes of everything. 
I used it when making sure the camera and the render context were aligned fine. 

Notice the `/**/` afterwards? 
It's for commenting the whole thing out by only adding a slash in front of the first line's `* {`.
I'll comment it out :)

Reply:
👍